gui.py

```diff
@@ -378,7 +378,7 @@ def __check_old_kontakt_version(self, kontaktdaten: dict = None) -> bool:
     #        Kontaktdaten        #
     ##############################
 
-    def kontaktdaten_erstellen(self, modus: Modus = Modus.TERMIN_SUCHEN):
+    def kontaktdaten_erstellen(self, modus: Modus = Modus.TERMIN_SUCHEN) -> bool:
```

Comment:
Bitte dann noch in den Docstring mit aufnehmen, dass man nun einen Bool bekommt. True bei Erfolg und False bei Abbruch